website/static/scripts/languageSelector.js

```diff
@@ -0,0 +1,13 @@
+function setLang(){
```

Comment:
@AngeloCG97 

nice way to solve this problem! 

My only comment is that changing HTML tags with JS may not be the best approach for SEO 

We can look into it and find out if this is a valid solution for google.

but it seems that Docusaurus is not flexible enough to support language tags natively